en/docs/references/api-path.md

```diff
@@ -0,0 +1,57 @@
+# API Path
+
+When you create a service/integration, you need to configure an API to invoke it. This configuration includes The **Path** field where you enter the path to be used by the API to trigger the service as well as to pass path parameters to it.
```

Comment:
Shall we use or instead of /.

Need to change the The to the in "The **Path** field".

Reply:
Corrected as specified.